scripts/cpp_lint.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python2
+#!/usr/bin/python
```

Comment:
According to http://stackoverflow.com/a/19305076/249226, the following would be recommended:

`#!/usr/bin/env python`

I'm not aware of any system where either shebang would fail. It looks like either should also work in Windows.